packages/matchbox/src/components/ThemeProvider/tests/ThemeProvider.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import ThemeProvider from '../ThemeProvider';
+
+describe('ThemeProvider', () => {
+  const subject = (props) => shallow(<ThemeProvider {...props} />);
+
+  it('should use the design system theme', () => {
+    expect(subject().prop('theme')).toMatchSnapshot();
```

Comment:
Not sure this snapshot offers much of value - it will get out of date when these values change, which will be a pain to manage in the long run.

Reply:
Yeah good point, I was on the fence about this, but I think a test like this should be in the tokens repo instead of here